src/components/NavDrawer.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import NavDrawer from './NavDrawer';
+
+describe('NavDrawer', () => {
+  it('should render correctly', () => {
+    const wrapper = shallow(<NavDrawer isOpen closeFunc={() => {}} />);
+
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should close when requested', () => {
+    const callback = jest.fn();
+
+    const wrapper = mount(<NavDrawer isOpen closeFunc={callback} />);
+    wrapper.find('Backdrop').simulate('click');
```

Comment:
I see that you're trying to find the Backdrop component that is used in the implementation of material-ui's Drawer component and click it to simulate a user clicking away from the navbar. Usually we don't want to rely on under-the-hood implementation details of dependencies since if material-ui changes the implementation of Drawer down the road to not use a Backdrop component anymore, this test will break. I think this test is fine for now since it's a small case, but it definitely deserves a comment.